astropy/io/ascii/tests/test_read.py

```diff
@@ -10,6 +10,7 @@
 
 import numpy as np
 import pytest
+from numpy.testing import assert_allclose, assert_equal
```

Comment:
There are still 16 calls to assert_equal in this file to clean up (some of them already pointed out by @dhomeier)
```suggestion
from numpy.testing import assert_allclose
```